src/app/modules/time-clock/store/entry.effects.spec.ts

```diff
@@ -339,4 +339,22 @@ describe('TimeEntryActionEffects', () => {
       expect(action.type).toEqual(EntryActionTypes.STOP_TIME_ENTRY_RUNNING_FAILED);
     });
   });
+
+  it('action type is UPDATE_ENTRY when UPDATE_TWO_ENTRIES executed', async () => {
```

Comment:
That was a standard for effect tests that we set with Rene, but I have already changed it in the first part so that it is better understood, without breaking the standard